beep/dataset.py

```diff
@@ -339,6 +339,87 @@ def serialize(self):
         return self.dataset_dir
 
 
+def get_threshold_targets(dataset_diagnostic_properties,
+                          cycle_type_target="rpt_1C",
+                          basis="discharge_energy",
+                          threshold=0.80,
+                          filter_kinks=None,
+                          extrapolate_threshold=True):
```

Comment:
If not `None`, what could `filter_kinks` be for example?

Reply:
I found for some of the cells a value of 0.04 seemed to be a good choice. Obviously depends a lot on what conditions you are trying to filter out.